docs/index.bs

```diff
@@ -1656,20 +1656,23 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
     <pre class="idl">
       [SecureContext, Exposed=(Window,Worker)]
       interface Cache {
-        [NewObject] Promise&lt;any&gt; match(RequestInfo request, optional CacheQueryOptions options);
-        [NewObject] Promise&lt;FrozenArray&lt;Response&gt;&gt; matchAll(optional RequestInfo request, optional CacheQueryOptions options);
+        [NewObject] Promise&lt;any&gt; match(RequestInfo request, optional MultiCacheQueryOptions options);
```

Comment:
Can you explain why methods on the Cache object would take a `cacheName` value via the `MultiCacheQueryOptions`?  It seems like the Cache object should be locked to itself and not be able to query other cache objects.

Reply:
Oh, I guess we always allowed `cacheName` at the webidl and would internally throw in those cases.

I guess I find it confusing to have a separate dictionary name with a more restrictive set of options, but not use it here.

Reply:
Agreed. I didn't want to change existing interfaces, but I guess this is backwards compatible. Will fix.

Reply:
Yes, this would be backwards compatible.